### PR TITLE
Allow to swap previous/next keys from the player

### PR DESCRIPTION
**Pull request by pa4**
Some bluetooth headphones have the prev/next button swapped

### CODE REVIEW

**Comment by tibbi** [comment]
do you have a headphone where you can easily reproduce next/prev being swapped?

**Comment by pa4** [comment]
I know it is a strange request but yea i unfortunately have such headphones. 
They are BASEUS Encok S06. I cannot find the user manual for it online unfortunately. 

They have + / - buttons and play/start button in the middle.
Short press on [-] decreases volume, [+] increases.
Long press on [-] sends next, and [+] sends previous key codes for whatever reason. This is confusing because you need to remember to press the "wrong" buttons for song change.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -68,6 +68,7 @@
 
         This app is just one piece of a bigger series of apps. You can find the rest of them at https://www.simplemobiletools.com
     </string>
+    <string name="swap_prev_next">Swap previous/next keycodes</string>
```
Comment (tibbi) [inline app/src/main/res/values/strings.xml]: add this to the settings section, just below "equalizer". Also add it to every language file, even if obviously not translated.

**Comment by tibbi** [comment]
alright, looks good. I just changed the settings string to make it less geeky and more understandable to people. Thanks

**Pull request merged**